src/main/java/com/cairnfg/waypoint/authorization/endpoints/household/ToggleActiveForHouseholdEndpoint.java

```diff
@@ -0,0 +1,99 @@
+package com.cairnfg.waypoint.authorization.endpoints.household;
+
+import com.cairnfg.waypoint.authorization.endpoints.ErrorMessage;
+import com.cairnfg.waypoint.authorization.entity.Account;
+import com.cairnfg.waypoint.authorization.entity.Household;
+import com.cairnfg.waypoint.authorization.service.AccountService;
+import com.cairnfg.waypoint.authorization.service.HouseholdService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.transaction.interceptor.TransactionAspectSupport;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@Tag(name = "Household")
+public class ToggleActiveForHouseholdEndpoint {
+
+  public static final String PATH = "/api/household/toggle-active/{householdId}";
+
+  private final AccountService accountService;
+  private final HouseholdService householdService;
+
+  public ToggleActiveForHouseholdEndpoint(HouseholdService householdService,
+      AccountService accountService) {
+    this.householdService = householdService;
+    this.accountService = accountService;
+  }
+
+  @Transactional
```

Comment:
So @Transactional is ok to use in various places. It will rollback all DB changes when an error occurs. However, note that this feature comes with overhead and can bog  down larger systems. For our uses here in Waypoints, it is fine but it can cause issues in large monoliths